crnk-testkit/build.gradle

```diff
@@ -1,11 +1,14 @@
-apply plugin: 'java'
+plugins {
+	id 'java'
+	id 'java-library'
+}
 
 dependencies {
-	compile project(':crnk-core')
-	compile project(':crnk-client')
-	testCompile project(':crnk-test')
-	testCompile project(':crnk-home')
+	implementation project(':crnk-core')
+	implementation project(':crnk-client')
+	testImplementation  project(':crnk-test')
+	testImplementation  project(':crnk-home')
 
-	compile 'org.assertj:assertj-core:3.9.1'
-	compile 'org.apache.httpcomponents:httpclient:4.5.2'
+	implementation 'org.assertj:assertj-core:3.9.1'
+	implementation 'org.apache.httpcomponents:httpclient:4.5.2'
```

Comment:
Should we move upto client5.httpclient or at least the latest version of 4.5.x

Reply:
This is wrapped up in the same changes as https://github.com/crnk-project/crnk-framework/pull/882#discussion_r1347473334. I think bumping up the the latest might be the right move, but haven't gotten too deep into that one yet.